network-config-analyzer/HelmScanner.py

```diff
@@ -0,0 +1,81 @@
+#
+# Copyright 2022- IBM Inc. All rights reserved
+# SPDX-License-Identifier: Apache2.0
+#
+
+"""
+Utility class to control HELM
+"""
+
+import os
+from CmdlineRunner import CmdlineRunner
+import re
+
+
+class HelmScanner:
+    """
+       A class for handling HELM and resolving HELM Charts
+    """
+
+    def __init__(self):
+        self.template_files = []
+
+    def parse_chart(self, chart_dir):
+        """
+        Resolve HELM Chart file and parse it for yamls.
+        :param chart_dir: The path of the Chart package
+        :return: dict: dict of yaml filenames and their resolved content
+        """
+        resolved_yamls_dict = {}
+        resolved_yamls = CmdlineRunner.resolve_helm_chart(chart_dir)[:-1]
+        resolved_yamls = str(resolved_yamls).encode('ascii', 'ignore').decode('unicode_escape')
+        resolved_yamls = resolved_yamls[:-1]
+        resolved_yamls_list = str(resolved_yamls).split('---')[1:]
```

Comment:
This solution is not good enough for splitting a yaml to document.
E.g., it will fail if there is a comment in the yaml that contains `---`.
A yaml parser must be used for this task